File: test-code/src/main/java/io/philo/test_code/app/Person.java
package io.philo.test_code.app;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

import static lombok.AccessLevel.PROTECTED;

@Entity
@NoArgsConstructor(access = PROTECTED)
@Getter
@ToString
public class Person {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(nullable = false)
    private String name;

    public Person(String name) {
        this.name = name;
    }
}

File: docker/src/main/kotlin/io/philo/docker/app/Person.kt
package io.philo.docker.app

import jakarta.persistence.*
import java.util.*

@Entity
class Person {


    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    var id: Long? = null

    @Column(nullable = false)
    var name = ""

    constructor() {
        this.name = UUID.randomUUID().toString()
    }
}